shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -9,86 +9,106 @@
-    public static final String CONTAINER_MERGE_SEGMENT_COUNT = "container_merge_segment_count";
-    public static final String CONTAINER_SEAL_COUNT = "container_seal_count";
-    public static final String CONTAINER_TRUNCATE_COUNT = "container_truncate_count";
-    public static final String PROCESS_OPERATIONS_LATENCY = "process_operations_latency";
-    public static final String PROCESS_OPERATIONS_BATCH_SIZE = "process_operations_batch_size";
-    public static final String OPERATION_QUEUE_SIZE = "operation_queue_size";
-    public static final String OPERATION_PROCESSOR_IN_FLIGHT = "operation_processor_in_flight";
-    public static final String OPERATION_QUEUE_WAIT_TIME = "operation_queue_wait_time";
-    public static final String OPERATION_PROCESSOR_DELAY_MILLIS = "operation_processor_delay_ms";
-    public static final String OPERATION_COMMIT_LATENCY = "operation_commit_latency_ms";
-    public static final String OPERATION_LATENCY = "operation_latency_ms";
-    public static final String OPERATION_COMMIT_METADATA_TXN_COUNT = "operation_commit_metadata_txn_count";
-    public static final String OPERATION_COMMIT_MEMORY_LATENCY = "operation_commit_memory_latency_ms";
-    public static final String OPERATION_LOG_SIZE = "operation_log_size";
-    public static final String ACTIVE_SEGMENT_COUNT = "active_segments";
-
-    // General metrics
-    public static final String CACHE_TOTAL_SIZE_BYTES = "cache_size_bytes";
-    public static final String CACHE_GENERATION_SPREAD = "cache_gen";
-    public static final String THREAD_POOL_QUEUE_SIZE = "thread_pool_queue_size";
-    public static final String THREAD_POOL_ACTIVE_THREADS = "thread_pool_active_threads";
+    public static final String CONTAINER_APPEND_COUNT = "segmentstore.container.append_count";
+    public static final String CONTAINER_APPEND_OFFSET_COUNT = "segmentstore.container.append_offset_count";
+    public static final String CONTAINER_UPDATE_ATTRIBUTES_COUNT = "segmentstore.container.update_attributes_count";
+    public static final String CONTAINER_GET_ATTRIBUTES_COUNT = "segmentstore.container.get_attributes_count";
+    public static final String CONTAINER_READ_COUNT = "segmentstore.container.read_count";
+    public static final String CONTAINER_GET_INFO_COUNT = "segmentstore.container.get_info_count";
+    public static final String CONTAINER_CREATE_SEGMENT_COUNT = "segmentstore.container.create_segment_count";
+    public static final String CONTAINER_DELETE_SEGMENT_COUNT = "segmentstore.container.delete_segment_count";
+    public static final String CONTAINER_MERGE_SEGMENT_COUNT = "segmentstore.container.merge_segment_count";
+    public static final String CONTAINER_SEAL_COUNT = "segmentstore.container.seal_count";
+    public static final String CONTAINER_TRUNCATE_COUNT = "segmentstore.container.truncate_count";
+    public static final String PROCESS_OPERATIONS_LATENCY = "segmentstore.process.operations_latency";
```

Comment:
here too

Reply:
btw, the name of the metrics should be `process_operations_***`; they relate to the `process operations` activity